web/src/global-styles/web-content.scss

```
@@ -0,0 +1,67 @@
+.web-content {
+    --link-color: #329af0;
+    font-weight: normal;
+    font-size: 14px;
+    line-height: 20px;
+    color: $gray-06;
+
+    h1,
+    h2,
+    h3 {
+        color: $gray-08;
+        font-weight: normal;
+    }
+
+    h4 {
+        color: $gray-06;
+        font-weight: normal;
+    }
+
+    h5,
+    h6 {
+        color: $gray-04;
+        font-weight: normal;
+    }
+
+    strong {
+        color: $gray-11;
+    }
+
+    small {
+        font-size: 12px;
+    }
+
+    &__search-keyword {
```

We're in `global-styles`, so global class names are okay :)
```suggestion
    .search-keyword {
```